antenna/configlib.py

```diff
@@ -0,0 +1,323 @@
+"""This module contains the configuration infrastructure allowing for deriving
+configuration from a .ini file and the process environment.
+
+In order of precedence:
+
+1. settings overrides (usually only applies when running tests)
+2. process environment
+3. .ini file (specified by ANTENNA_INI environment variable)
+
+Example of usage::
+
+    from antenna.configlib import config
+
+    DEBUG = config('DEBUG', default=True, parser='bool')
```

Comment:
Two things:

First, this example is wrong. I need to fix it. It should be:

```
DEBUG = config('DEBUG', default='True', parser=bool)
```

The default is always the text version of the value--it's what you'd put in the environment, the ini file, etc.

Second, this structure makes it easier to encode complex values like db urls in the code. That way the parser has the job of breaking it out.

For example, what parser would this use?:

```
DATABASES = config('DATABASES', default='postgres://foo:bar@baz.com:5432')
```